snc.sh

```diff
@@ -17,10 +17,10 @@ SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_c
 # If user defined the OPENSHIFT_VERSION environment variable then use it.
 # Otherwise use the tagged version if available
 function get_openshift_version {
-    if [ ${OPENSHIFT_VERSION} != "" ]; then
+    if [ "${OPENSHIFT_VERSION}X" != "X" ]; then
         OPENSHIFT_RELEASE_VERSION=$OPENSHIFT_VERSION
     else
-        OPENSHIFT_RELEASE_VERSION=$(git describe --exact-match --tags HEAD 2>/dev/null)
+        OPENSHIFT_RELEASE_VERSION=$(git describe --always --tags HEAD 2>/dev/null)
```

Comment:
@cardil this is by design since we don't want to run a tagged version of release so if we merge this then there will be no difference between the master branch or a tagged version.

```
$ git branch 
* master

 $ git describe --exact-match --tags HEAD
fatal: no tag exactly matches '2c2dc7d22758cbe39df1411965d0ed6f1b574165'

$ git checkout 4.1.14 <= or any tag/releases
$ git branch 
* (HEAD detached at 4.1.14)

$ git describe --exact-match --tags HEAD
4.1.14
```

Reply:
It will be different:

```
$ git branch 
* master

 $ git describe --always --tags HEAD
4.1.14-8-g037212f

$ git checkout 4.1.14
$ git branch 
* (HEAD detached at 4.1.14)

$ git describe --always --tags HEAD
4.1.14
```

On tagged commit, it will be equal to tag. On other commits it gives a meaningful output.
The current head is based on `4.1.14`, but since it has a few commits on top of that, describe has added the number of additional commits `8` and an abbreviated object name for the commit itself `037212f` at the end.

In that way it's always unique and easy to know is that was official release of build of something else.

Reply:
I removed it, as I didn't saw how this is used. Sorry.